template.yaml

```diff
@@ -7,11 +7,95 @@ Parameters:
     Type: String
     Default: "/SungardAS/Alerts/Account"
     Description: Account Cloudwatch Log Group Name
-  SubscriptionFilterDestinationArn:
+  DestinationArn:
     Type: String
     Description: Subscription Filter Destination Arn
+  DbUserName:
+    Type: String
+    Description: user name of mcaws db
+  DbPassword:
+    Type: String
+    NoEcho: true
+    Description: password of mcaws db
+  DbHost:
+    Type: String
+    Description: Host of mcaws db
+  MasterMgmAwsId:
+    Type: String
+    Description: AWS Id of Master Management Account
+  AdminRoleName:
+    Type: String
+    Description: AWS admin role name
+  DataDogAwsId:
+    Type: String
+    Description: Aws Id of Datadog Aws Account
+  BillingApiKey:
+    Type: String
+    NoEcho: true
+    Description: Access key of Billing System
+  BillingSecretKey:
+    Type: String
+    NoEcho: true
+    Description: API key of Billing System
+  BillingServer :
```

Comment:
we should have said BillingEndpoint instead.

Reply:
If I am going to change just name then i need to change a lot of places 
its required to again test with all scenario

Reply:
ok do not change it.